Envoy failed on Bazel CI with "unused variable 'factory_name'"

https://buildkite.com/bazel/envoy/builds/1295#b92e2abd-a008-49b7-a929-2a5dde5a0152
```
bazel-out/k8-fastbuild/bin/include/envoy/registry/_virtual_includes/registry/envoy/registry/registry.h:348:44: error: unused variable 'factory_name' [-Werror=unused-variable]
     for (const auto& [factory_name, factory] : factories()) {
                                            ^
cc1plus: all warnings being treated as errors
```

bazel-out/k8-opt/bin/include/envoy/registry/_virtual_includes/registry/envoy/registry/registry.h:348:44: error: unused variable 'factory_name' [-Werror=unused-variable]
     for (const auto& [factory_name, factory] : factories()) {

--- a/.bazelci/presubmit.yml
+++ b/.bazelci/presubmit.yml
@@ -1,10 +1,5 @@
 ---
 tasks:
-  gcc:
-    name: "GCC"
-    platform: ubuntu1804
-    build_targets:
-      - "//source/exe:envoy-static"
   rbe:
     name: "RBE"
     platform: ubuntu1804

